spec/features/user_sees_list_of_books_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature 'User sees a list of books' do
+  scenario 'successfully' do
+
+    books = create_list(:book, 3)
```

Comment:
You usually want to create as few records as possible. In this case, that would be `2`. There is a nifty method called `create_pair` that will create 2 of them for you